python/ray/serve/_private/http_proxy.py

```diff
@@ -305,23 +312,38 @@ def _update_routes(self, endpoints: Dict[EndpointTag, EndpointInfo]) -> None:
 
         self.prefix_router.update_routes(endpoints)
 
-    def _update_draining(self, active_nodes: Set[str]):
+    def is_drained(self):
+        if not self._is_draining():
+            return False
+
+        return (not self._ongoing_requests) and (not self._draining_timer)
```

Comment:
I think the draining timer is completely unnecessary -- can't you just check if `time.time() - self._drain_start_time > MIN_DRAIN_PERIOD`?

Reply:
oops, yea, its no longer needed.